src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int COUNT_BALL = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery[] lotteries = new Lottery[COUNT_BALL];
+        for (int i = 0; i < lotteries.length; i++) {
+            lotteries[i] = new Lottery();
```

Comment:
Why do you create an array of Lotteries? We need ONE Lottery. And we can call 3 times getRandomBall() from it.